blocks/leadspace/leadspace.css

```diff
@@ -1,4 +1,36 @@
 /* Leadspace block styles */
+.leadspace, .leadspace p {
+    font-family: var(--sans-serif-font-light);
+    font-size: var(--font-size-16);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-160);
+    letter-spacing: var(--letter-spacing-1);
+    }
+
+.leadspace h1, h2 {
+    font-family: var(--serif-font);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-120);
+    letter-spacing: var(--letter-spacing-1-75);
+    }
+
+.leadspace h1 {
+    font-size: var(--font-size-36);
+    margin: var(--spacer-layout-02) 0;
+    }
+
+.leadspace h2 {
+    font-size: var(--font-size-32);
+    margin-bottom: var(--spacer-layout-02);
+    }
+
+.leadspace h4:first-of-type {
+    font-size: var(--font-size-11);
```

Comment:
Should we force uppercase too?

Reply:
I hemmed and hawed over this. I decided to leave it without uppercase, because I think there were statistics in one of the Figma designs that weren't all uppercase but had the same treatment.